declarations/Mektoube.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Mektoube",

```

Comment:
Few adjustements in https://github.com/OpenTermsArchive/services-dating/pull/2/commits/447bf293cd943269d9ddebf266069a8034cf6f3f